### DISCUSSION TITLE
Natural Process

### DISCUSSION THREAD

**Discussion by neilotte** [CommonCoreOntology/CommonCoreOntologies#398] [Scope Questions]
The present definition holds: "A Process existing in or produced by nature; rather than by the intent of human beings."

It is unclear how to interpret this class or its subclasses. 

Taking either side of the semi-colon: 

I don't know what 'A process existing in or produced by nature' means. Does this mean, for instance, that any type of event that would have instances regardless of whether or not humans exist would be a natural act? This would seem to make too many kinds of process fall under cco:NaturalProcess rather than cco:IntentionalAct. For instance: violence, nursing young, going to war, play, sex, tool use... 

Taking the second part of the semi-colon: 

Right now, 'rather than by the intent of human beings' reads to me as if every cco:NaturalProcess and its subprocesses must be unintentional. But some births, some deaths, some spinning motions, some propulsion processes, etc.... are intentional. 

Thoughts?

**Comment by nklsbckmnn** [comment]
Not sure if that solves anything, but in the examples you give below (some births etc.) one could say that there exists some intentional act causally upstream from some natural process or mechanical process (if we are talking of spinning motions, propulsion processes of artifacts). So an "intentional birth" would be two processes. That would leave the question what the genus of that compound process should be… maybe just process.

I wonder about "unintentional act" and "behavior“… those classes seem not to be distinct from natural processes.

I agree that the part of the definition on the first side of the semi-colon taken by itself is too broad.

**Comment by neilotte** [comment]
@eliasweatherfield That's a good thought regarding upstream causation, but I don't know if I buy it. This seems like another place where invoking causation is generally a bad idea, since it frequently requires an intransitive demarcation of causes and effects that can strike an audience as capricious (e.g. "Judge, I didn't cause that kid to be born. I'm was just the upstream dad. His birth was a natural process brought about by nature"). 

Also, while we're on the topic of birth, this leaves one without a good place for caesarean delivery. It seems one would have to say that during a caesarean, there are two co-occurring processes that involve exactly the same participants: the birth and the caesarean delivery--the former a natural process, the latter an intentional act.

**Comment by nklsbckmnn** [comment]
I would argue that a caesarean delivery is distinct from a natural birth. It is a birth only insofar as birth is a defined class the result of which is that a child has left the womb.

I opened an issue here some time ago about whether the agent in relation requires that an agent is causally active within a process or if causal activity upstream is sufficient. I personally came to the conclusion that the former interpretation is more useful. But your doubts about demarcation seem to apply to that question too, don't they?

But I admit there might be processes where it is not enough to say that there was an intentional act upstream because there seem to be intentional acts in the course of the "natural process" as well. It would be strange to regard multiple intentional acts of pushing by the mother during birth as seperate from that birth. I think that's what you were getting at, wasn't it?

**Comment by neilotte** [comment]
To the questions above, yes and yes. And if cco:Birth is being used only for 'natural births', then agreed caesarean delivery would be distinct (though, in practice, I believe its being used to anchor all birthdates, whether brought about by a mother's pushing, foreceps, ceasearean, or other agent-intended acts--could be wrong here). 

To the larger point: Natural Process remains troublesome :)

**Comment by nklsbckmnn** [comment]
I guess a simple (probably too simple) solution would be to say that human intentionality in a natural process is not necessary.

**Comment by aellenhicks-jhuapl** [comment]
In a correspondence with @rorudn I asked about Explosion. 

"(AH) Explosion - Some, but not all, explosions are produced by the intent of human beings. Should it be a subclass of Natural Process? - (RR) No. It should be a subclass of BFO Process."

**Comment by nklsbckmnn** [comment]
Allow me to try to make some progress on this issue.

It seems to me that if we understand a CCO:Natural Process as a Process that can have instances that are neither (a) caused by some Act nor (b) need to be sustained by some parallel/intermittent Act that would still allow us to represent the following:

(1) An Act or an Intentional Act is cause of a Natural Process.
(2) A Natural Process is process part of an Act or Intentional Act.
(3) An Act or Intentional Act is process part of a Natural Process.

A case exemplifying (1) and (2):

An explosion is intentionally caused by a person.

Intentional Explosion is an Intentional Act that has two process parts:
(a) Induction of Explosion which is an Intentional Act
(b) Explosion which is a Natural Process and caused by Induction of Explosion

I am aware this might look tedious. But it seems to me to be at the level of granularity required e. g. to represent cases in criminal law.

A case exemplifying (3):

An extrasolar object that is hurtling towards earth is successfully deflected by an Intentional Act.

The Motion of the extrasolar object through the solar system (from entering to leaving it) is a Natural Process.
The "Act of Deflection" in an Intentional Act that is process part of that Motion.

I would appreciate any comments.

**Comment by nklsbckmnn** [comment]
I just realized that because of the causative role that an Agent plays in the Motion of the extrasolar object, that Motion would also be an Act... 

Can we accept that Natural Processes can also be Acts? Act would need to become a defined class to allow this.

**Comment by alanruttenberg** [comment]
In discussion in the MLO meeting I think it was @mark-jensen  who said something to the effect that the terms was added for pragmatic reasons - to make it not be a huge list of sibling processes. Please correct me if I got this wrong. 
My suspicion is that with some analysis we can introduce a hierarchy that addresses the pragmatic issue but with more defensible terms than "natural x"

**Comment by APCox** [comment]
@eliasweatherfield Your elucidation is in line with how I have been using the 'Natural Process' versus 'Act' distinction. The idea is that a combustion process, for example, is a combustion process regardless whether the proximate cause of the combustion was a lightning strike or an intentional act of striking a match. There is no ontological difference between the 2 fires (apart, perhaps, from the processes that caused them). Hence, in both cases the combustion process should be represented as "natural" -- it's chemistry in action. Nonetheless, in everyday parlance we say imprecise things such as, "Fires may be natural or man-made."

Causation is a complex and vastly overloaded topic. Beyond the basic fact that (almost?) no event has a single "cause", how we think about causation is typically closely tied to legal and moral issues (we always want to know who to blame or praise and will go to great lengths to do so -- for example, see Neil's comment above about "the upstream dad"). Differing understandings of causation are one way that things can go wrong here.

The CCO only addresses causation in generalities and care should be taken to not read too much into statements such as:
`Act =def "A Process in which at least one Agent plays a **causative** role."`
The definition is insufficient to handle all edge cases, but hopefully is good enough to provide sufficient guidance for its usage in most cases. The intent behind this definition is to pick out those processes that are clearly **_always_** agential (e.g., coughing, which could be intentional or unintentional) from those that are not (e.g, ice melting). 

Of course, this does not prohibit me from causing an ice melting process to occur by performing one of a variety of acts. It is important to note, however, that the ice melting process I caused is still a natural process -- at least according to the **_intended_** sense of cco:NaturalProcess.

Another way that things can go wrong here is, as Elias notes, by approaching the same thing from different levels of granularity. For example, one person may represent 'Arson' as "Intentionally and unlawfully burning a building down." while another person represents it as "Intentionally and unlawfully starting a fire." The second person separates the burning process from the fire lighting process, whereas the first person combines them into a single process that must be placed under 'Intentional Act'.

The CCO attempts to enable representations at multiple levels of granularity and therefore needs to make these sorts of distinctions. This is one reason why, e.g., 'Birth' and 'Death' are under 'Natural Process'. @neilotte's example of a caesarean delivery is, I think, a good example of how an Act -- the caesarean delivery -- causes a Natural Process -- the birth. I fail to see an issue here. Both 'Caesarean Delivery' and 'Natural Birth' (in which it should be noted that the word 'natural' is used in a different sense than in cco:NaturalProcess) are "larger" or "thicker" processes than 'Birth' is. Hence, an instance of 'Birth' may be part of or caused by an instance of 'Caesarean Delivery', 'Natural Birth', or another process. Both 'Caesarean Delivery' and 'Natural Birth' reasonably belong under 'Intentional Act', but that does not necessitate 'Birth' to be there as well. I understand why someone might choose to build a hierarchy in which 'Caesarean Delivery' and 'Natural Birth' are subtypes of 'Birth'; however, it is worth considering the parallel case of death. If the same approach were taken with 'Death', then 'Murder' and 'Natural Death' would both be subtypes of 'Death'. But that would make 'Natural Death' an 'Intentional Act', which is surely wrong! It is better, I contend, to separate the cause of death (the act of murdering, the heart attack, the disease course, etc.) from the death itself, which is always a 'Natural Process'. The same reasoning holds for a great many other processes.

Now, with all of that being said, there is a LOT to be desired in the current definitions of 'Act' and 'Natural Process', more so for the latter. In the case of 'Act', the biggest challenge is properly defining the nature and limiting and scope of causation that makes a process an Act. This is not an insignificant task. Improving the definition of 'Natural Process' is, I think, more tractable; however, I suspect that doing so will also require a better account of the sort of causation involved.

@alanruttenberg I stated during the MLO call today that 'Natural Process' was added as a complimentary class to 'Act' at least in part for the pragmatic purpose of better organizing the subtypes of bfo:process. Perhaps the class is not strictly necessary and I'm happy to discuss alternative solutions. I suspect, however, that most people's concerns can be alleviated by improving the definition(s) and better documenting how the terms should be used.

**Comment by nklsbckmnn** [comment]
@APCox Thank you for your insights.

> The intent behind this definition is to pick out those processes that are clearly **_always_** agential

If I understand correctly, this would enable me to not require the asserted instance of the Natural Process of Motion in my extrasolar object case to also be (inferred to be) an Act despite having an Intentional Act as part. And one could even have Natural Processes that have some Agent (CCO:Natural Process CCO:has agent some CCO:Agent).

I really like that solution.

**Comment by APCox** [comment]
@eliasweatherfield Yes, that is the idea.

**Comment by alanruttenberg** [comment]
I don't think you'll be able to define natural process , based on my prior experience with attempts to do that in OBO Foundry. It doesn't help much to appeal to causality, "thicker" or "thinner" processes, as those things don't have obvious meaning and so don't really help in understanding. Causality is really hard to work with as judgement of whether something is a cause is hard and subject to disagreement. So relying on people making such judgement will hurt interoperability. 

I think there might be more luck in defining intentional act. One thing I don't see in the definition is whether the agent with the intention is necessarily  participates in the process. That might help scope things some.  I also note the language "plays a role" in the definition which isn't the sense in which BFO uses roles, There's no playing a role. There is realizing a role, and realizing a role requires participation of the bearer in the process.

I'm working on a more comprehensive comment that I hope will demonstrate more of the problems.

**Comment by mark-jensen** [comment]
I don't like distinctions based on use of terms like 'natural', 'artificial', 'real, and so on. Contention almost always ensues. Intuitively I can certainly understand the desire to make the distinction. There are many such processes that happen independently of animal actions, such that do not require a cognitively endowed agent to initiate or actively maintain or regulate. The list is quite long: meteorological, geological, chemical, most biological ones, and so on and on and on. But of course some of these can be caused and regulated by agents too... 

Without losing focus over the concern about use of 'natural', let's just consider what distinction is attempting to be made with the term `Natural Process` (NP).

@alanruttenberg asks if 
> whether the agent with the intention is necessarily participates in the process [an Intentional Act]

I have always assumed that the answer to that is a firm yes. The clause in the definition, "prescribed by some Directive Information Content Entity held by at least one of the Agents" implies to me that a Plan has been adopted by a participant with intention to carry it out. Moreover, Acts also require agents as participants. Unless playing a causative role allows doing so from a spatio-temporal distance that rules out participation. X has_agent Y is a sub of participation where Y is casually active in X.

@eliasweatherfield states and @APCox seems to agree
> And one could even have Natural Processes that have some Agent (CCO:Natural Process CCO:has agent some CCO:Agent)

I simply do not see how that can be the case. Isn't the intent behind NP to differentiate processes which specifically do not have participants that act as agents in the process? The definition implies as much
> existing in or produced by nature; rather than by the intent of human beings

If possible to have `NP has_agent X`, then how is NP not also an Act ("a process in which an agent plays a causative role")? I can't recall now why CCO:Act doesn't have the axiom `has_agent some entity`. 

Providing Acts do require at least one participant acting as an agent (Note: this is stronger than merely saying a thing which is also an agent is a participant), then it is plausible to have a class like NP which is complement to Act and defined by strictly ruling out agential participation.

`AgentlessProcess =def A Process that does not have an agent as an active participant`

Forgive the cruddy label, I'm sure something better can be used. At this point I don't see another way to coherently distinguish what is now an NP from an Act that will be clear to users of CCO. This isn't a recommendation for keeping the term and making the distinction, just initial thoughts to stir discussion.

If it's decided to keep a term like NP, an improved definition won't necessarily resolve some of the potential problems with subclasses of the current NP, such as reconciling (natural)Motion vs. Act Of Motion, or needing intentional variants of Ignition, Propulsion, Wave Production, or Birth. If folks look closely at some these and consider their use cases, it may become clear that having a term like NP, as in agentless, is simply not that helpful and potentially more trouble that it's worth.

**Comment by alanruttenberg** [comment]
Regarding @eliasweatherfield's
   > And one could even have Natural Processes that have some Agent (CCO:Natural Process CCO:has agent some CCO:Agent)

One example: Is creation of an anthill a natural process or an intentional act? Do ants have intention? The definition of agent allows for animals to have them. If ants are considered not to have intention then where is the line drawn. If they are considered to have intention then it answers an earlier question of whether natural processes are processes that wouldn't happen if there were no people. The answer is "No".

@neilotte in the OP
> I don't know what 'A process existing in or produced by nature' means. Does this mean, for instance, that any type of event that would have instances regardless of whether or not humans exist would be a natural act?

What about unintended side effects of drugs? Seems like a natural process, but there's a reasonable argument to be made the the doctor prescribing or the patient taking the drug is a cause and so they would be considered agents.

I think pretty much any unintended result of intentional action suffers this problem., because of the definition of  has_agent:
> x has_agent y iff x is an instance of Process and y is an instance of Agent, such that y is causally active in x.

Set aside the problem of determining whether something is causally active, and that *is* a problem, the cases I give seem to satisfy the definition, because from Agent you only have that the entity is *capable* of intentional acts, but the has_agent relation doesn't say that the act *is intentional*. It only says *causally active*. Just because something is capable of intentional acts doesn't mean that all their acts are intentional. But there are lots of unintentional acts where one would  judge that there was a party that was causally active in the unintentional process.

@mark-jensen's offer
>  AgentlessProcess =def A Process that does not have an agent as an active participant

It's generally considered poor practice to define a term negatively. I suppose it's ok if we're talking about a defined class. But even then I'm not sure it would be desirable. You are going to also have trouble determining who are active participants. By which I mean although there are cases where it seems clear, there will also be cases where it's hard to make the assessment.

I'm looking at the environs of Natural Process and other things stand out. Death is sometimes natural sometimes not, as when a prisoner is executed, when it involves deliberate ingestion of substances with the aim to end life. Or when a soldier is blown up by an IED.  Yes, you can say that the death is a separate process from the processes I describe, which are only preliminary, but I think it's hard (the IED explosion is hard to think of as separate) and not obviously useful. 

As to the motivation to be able to organize terms, birth and death are parts of a person's history (BFO sense). A class part of history, might be be better grounded. Or radio interference. I don't think you can separate the interference from the jamming. If the jamming stops, the interference stops. jamming *is* generating interference. And would the term satisfy the definition of mechanical process? As an aside that definition looks too broad. Similarly the wave terms seem like mechanical processes.

Intention is some kind of realizable entity. Given that I wouldn't have intentional processes (or it's complement) be part of the asserted hierarchy. The usual pattern is to augment the primary hierarchy with defined classes whose placement in the hierarchy is inferred.  A doctor is a person with a doctor role. A health care encounter with a doctor is a realization of the doctor role.  An intentional process is a defined class defined as a process that realizes the intention. 

This pattern avoids the kinds of troubles we are having sorting processes into intentional and unintentional. It's perfectly fine for some explosions to be realization of an intention, and others not.

BFO's only mechanism for capturing (some) causal contribution is the realizable entity. Because a realization is preceded by the acquisition of the realizable entity, and correlated with the realizable entity, and because the bearer participates in the realization,   it is, loosely speaking, reasonable to consider the bearer as causally active in the realization.

**Comment by mark-jensen** [comment]
@alanruttenberg From BFO perspective, can clear guidance be given on the bounds of participation?

Any view that says a Dr is a participant in some adverse biochemical reaction in a person's body after a drug is ingested seems ludicrous. Is asserting distal participation like that allowable? More importantly, does it make any sense?

`has_agent` in CCO is participation that is both causal and active. So an agent has to participate in any process they are casually active in.

While you might convincingly argue that the Dr had some "causal role" in the adverse reaction, I can't see how they would be a casually active participant in said process. Even if they injected the drug. The Dr was casually active (agent) in the process of prescribing or injecting, but certainly not an agent in the adverse reaction.

FYI, I have always operated under the assumption that use of "plays a causative role" in the definitions for Act and Intentional Act require direct active participation by the agent. A trapper that sets a snare is not agent in the release of the snare weeks later.

**Comment by nklsbckmnn** [comment]
@mark-jensen 

> I have always assumed that the answer to that is a firm yes. 

Agreed.

> The clause in the definition, "prescribed by some Directive Information Content Entity held by at least one of the Agents" implies to me that a Plan has been adopted by a participant with intention to carry it out.

Maybe a little off-topic. But I have a problem with this. Let's say a CCO:Social Act has two participants. It has two process parts each corresponding to an Intentional Act by one of the participants. I guess being agent in a subprocess implies being agent in the superprocess. But why is it necessary that one of the participants also has a Directive ICE prescribing the Social Act as a whole? Is it not enough that he intents his part of the Social Act? Intending part of an Act does not mean intending the whole Act, does it?

> If it's decided to keep a term like NP, an improved definition won't necessarily resolve some of the potential problems with subclasses of the current NP, such as reconciling (natural)Motion vs. Act Of Motion, or needing intentional variants of Ignition, Propulsion, Wave Production, or Birth. If folks look closely at some these and consider their use cases, it may become clear that having a term like NP, as in agentless, is simply not that helpful and potentially more trouble that it's worth.

Don't we want some Entity that is common to Act of Motion and (natural) Motion? What could that be in your opinion? One suggestion would be: A Motion that is not asserted as an Act but still can have agent some Agent. If agency is sufficient for an CCO:Act, the Motion could be inferred to be an Act. If necessary agency is required fo an CCO:Act, the Motion would have agent some Agent without being an Act.

**Comment by nklsbckmnn** [comment]
@mark-jensen 

> While you might convincingly argue that the Dr had some "causal role" in the adverse reaction, I can't see how they would be a casually active participant in said process. Even if they injected the drug. The Dr was casually active (agent) in the process of prescribing or injecting, but certainly not an agent in the adverse reaction.

> FYI, I have always operated under the assumption that use of "plays a causative role" in the definitions for Act and Intentional Act require direct active participation by the agent. A trapper that sets a snare is not agent in the release of the snare weeks later.

Agreed. I made a similar argument in #79

That's why I think the problematic cases are those in which a Natural Process is sustained or influenced by some parallel/intermittent Act, not the ones where an Act can be said to cause some Natural Process.

**Comment by alanruttenberg** [comment]
@mark-jensen 
>From BFO perspective, can clear guidance be given on the bounds of participation?

Not much. Processes occupy temporal and spatiotemporal regions which can rule some some things out. There's some thing about how gdcs and sdcs can participate in processes - if they do then their bearers do too. For histories there's a precise definition and an assertion that the thing whose history it is participates in it at all times.

But it's not always clear to me what spatiototemporal region of a process is. And there are disagreements over things like the rotating ball that is heating up. Barry sees these as two processes - a heating process and a rotating process -that coincide spatiotemporally. I see one process and a need to be able to logically express the relation of those two to the single process. I think that some sort of process dependents are needed.

One of my beefs about process profiles is that they are part of a process but I have no clue what spatiotemporal region they occupy or what participates in them. Barry calls them "thin processes" and I really have no clue what he's talking about, as all process occupy a spatiotemporal region. 

There's also a sense that process and their boundaries are kind of fiat.

I would love to be able to make this clearer, but don't know how to at the moment.

**Comment by alanruttenberg** [comment]
@mark-jensen: from a CCO perspective, can clear guidance be given on the bounds of causal activity? ;-)

**Comment by mark-jensen** [comment]
@alanruttenberg 1.21 Gigawatts!

**Comment by BrianHaugh** [comment]
Some of the earlier comments seem to confuse types of processes with instances. Many types of processes, such as "Wave Processes" can have both Natural and "Artificial" (intentional) instances.  Wave Processess include the natural electromagnetic output of our Sun and other stars, as well as the intentional artificial processes generated by radio and TV broadcasts. 
Recommend that Wave Processes be moved to be a direct subclass of obo:Process. 
Recommend that we retain the class "cco:Natural Process" since it is a useful distinction even though many, if not all of its current subclasses need to be moved since they have both natural and intentional instances.

**Comment by alanruttenberg** [comment]
In the MLO meeting today we discussed that there were natural processes that could both be both intentional and natural/not intentional. I propose that we remove this distinction in the asserted hierarchy, and then add a way of having additional restrictions that capture the intentional/natural distinction. For instance suppose we had a property processkind with values intentional or unintentional.(that isn't a proposal - just a way to explain the issue). Then, for each process that you want to make this distinction on, you add a restriction 'and processkind value intentional' or 'and processkind value unintentional' or 'and (processkind value intentional) or (processkind value unintentional)' 

Then there are two defined classes that are made. (process and processkind value unintentional) sometimes called 'natural process', and (process and processkind value intentional) sometimes called act or intentional process.

The rest of the processes will then be inferred to be subclasses of one or both of the above defined classes, and instances may assert a value for processkind, if known.

TBD: Make a proper proposal for how to capture the distinction.

**Comment by BrianHaugh** [comment]
CCO already has a cco:IntentionalAct class.  Identifying an instances of a processes that needs to be identified as intentional is readily accomplished by asserting it to be a type of cco:IntentionalAct.   Since there are many useful subclasses of Intentional Act, I recommend retaining this class and its subclasses. 
So, we really only need another way to distinguish NaturalProcesses if we want to be able to distinguish them from Intentional Processes. I consider this a useful distinction, especially as applied to SETI,  which requires distinguishing artificial from natural electromagnetic processes :-). Also a concern in seismology when distinguishing nuclear blast seismic events from natural ones. I suggest just retaining the coo:Natural Process class, but moving its subclasses up a level as direct subclasses of bfo:BFO_0000015. If retaining cco:NaturalProcess, we would do well to define it more precisely, e.g., to exclude mixed Natural/Intentional processes, as well as to address the causal proximity requirements for intentional processes.

**Comment by alanruttenberg** [comment]
Maybe. IntentionalAct would still need to be a defined class so that instances of processes that can be both are properly classified when intentional.

I'm also not sure whether some of the hierarchy for intentional act is organized in such a way that it or part of it would be desirable to be applicable to both kinds of process. 

My gut is to not do it at this level. It may turn out that some of the subtrees are organized in a way that don't admit a second classification. I'd look at it case by case. If we're using an alternative way of representing intentional vs other acts, then our default should be use that uniformly and use a primary classification that is independent of that aspect. 

A representative question: There is motion and Act of motion and it's subclasses. If we go with the proposal these two might gain a common parent. If act of motion had the current asserted parent we would now have a case of asserted multiple inheritance, something that we try to avoid. Many of the communication processes have accidental counterparts - cases where there was no intention to communicate but, say, a microphone switch was left on by mistake, or someone accidentally hits the send button before they are finished writing.

It's tricky. Is walking always intentional? What about sleep walking? Manslaughter vs murder? Criminal acts where the perpetrator was unaware of the law? Entertaining events that aren't planned, like being amused by the behavior of passersby? To find some of these we can look at a definition like this

> An Intentional Act consisting of any activity which provides a diversion or permits people to amuse themselves.

Strike out the "An intentional act" part, use the rest of the definition, and see whether there are instances that aren't intentional but satisfy the definition.

**Comment by neilotte** [comment]
Moving to Discussion.
